packages/gatsby/src/commands/build-html.ts

```diff
@@ -1,18 +1,21 @@
-/* @flow */
-const Promise = require(`bluebird`)
-const webpack = require(`webpack`)
-const fs = require(`fs-extra`)
-// const convertHrtime = require(`convert-hrtime`)
-const { chunk } = require(`lodash`)
-const webpackConfig = require(`../utils/webpack.config`)
-const reporter = require(`gatsby-cli/lib/reporter`)
-const { createErrorFromString } = require(`gatsby-cli/lib/reporter/errors`)
-const telemetry = require(`gatsby-telemetry`)
+import Bluebird from "bluebird"
+import webpack from "webpack"
+import fs from "fs-extra"
+// import convertHrtime from "convert-hrtime"
```

Comment:
Is there a reason you've left this in?

Reply:
It was in there before, so I figured it was left for a reason /shrug